alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/pom.xml

```diff
@@ -32,7 +32,7 @@
     <acs.search.api>${project.build.directory}/definitions/alfresco-search.yaml</acs.search.api>
     <acs.search-sql.api>${project.build.directory}/definitions/alfresco-search-sql.yaml</acs.search-sql.api>
 
-    <java.version>21</java.version>
+    <java.version>17</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
```

Comment:
These versions shouldn't be needed here. There are versions specified in main pom and they seem to have been changed correctly. The properties here are redundant, as they should be inherited from the main pom.xml file from alfresco-java-sdk.

Reply:
Yes removed the redundant property.

Reply:
Remove also
    <maven.compiler.source>${java.version}</maven.compiler.source>
    <maven.compiler.target>${java.version}</maven.compiler.target>
    
Those are defined in main pom.xml too.

Reply:
Removed the below property from - alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib/pom.xml
<maven.compiler.source>${java.version}</maven.compiler.source>
<maven.compiler.target>${java.version}</maven.compiler.target>